frameworks/CSharp/aspnetcore/PlatformBenchmarks/Program.cs

```diff
@@ -55,13 +51,20 @@ public static IWebHost BuildWebHost(string[] args)
             Console.WriteLine($"Database: {appSettings.Database}");
             Console.WriteLine($"ConnectionString: {appSettings.ConnectionString}");
 
+    #if NPGSQL
```

Comment:
Why not just `#if DATABASE` ?

Reply:
Do you mean to get rid of the individual library preprocessor directives here and just use one check for all/both of them?
Probably simpler.
I implemented it the way it currently is, as a sanity check for developers when copy pasting docker files, so they get a message that is a bit more helpful than "cannot connect" or so, but the code already outputs the connection string, so this might be overkill.